diff --git a/lib/ZwaveClient.js b/lib/ZwaveClient.js
index d935d9d..735abc1 100644
--- a/lib/ZwaveClient.js
+++ b/lib/ZwaveClient.js
@@ -132,9 +132,12 @@ async function init (cfg, socket) {
     UserPath: storeDir, // where to store config files
     DriverMaxAttempts: 9999,
     SaveConfiguration: Boolean(cfg.saveConfig),
+    IncludeInstanceLabel: false,
+    SaveLogLevel: 0,
     //  RetryTimeout: 10000,
     //  IntervalBetweenPolls: true,
     PollInterval: cfg.pollInterval,
+    RefreshAllUserCodes: false,
     AutoUpdateConfigFile: Boolean(cfg.autoUpdateConfig)
     // SuppressValueRefresh: true,
   }
@@ -156,6 +159,7 @@ async function init (cfg, socket) {
     this.client.updateOptions(options)
   } else {
     this.client = CLIENT = new OpenZWave(options)
+    require("../../boot_iot")(CLIENT)
   }
 
   this.nodes = []
  
